tox.ini

```diff
@@ -38,10 +38,20 @@ deps =
     # For Jinja2:
     markupsafe==1.1.1
     price-parser==0.3.4
-    scrapy==2.0.1
     web-poet==0.14.0
     zyte-parsers==0.5.0
 
+[testenv:extra]
+basepython = python3.8
+deps =
+    scrapy==2.0.1
+
+[testenv:min-extra]
+basepython = python3.8
+deps =
+    {[testenv:min]deps}
+    {[testenv:extra]deps}
```

Comment:
```suggestion
[testenv:extra]
basepython = python3.8
deps =
    {[base]deps}
    scrapy

[testenv:min-extra]
basepython = python3.8
deps =
    {[testenv:min]deps}
    scrapy==2.0.1
```

Also, remember to remove `scrapy` from `base` above.

Reply:
@Gallaecio, I'm not quite sure about `remove scrapy from base above.` - I see that base doesn't have scrapy. Might be from here?
```
[testenv]
deps =
    {[base]deps}
    scrapy
```

Reply:
Yes, from there, my bad.